sbom-utility-scripts/scripts/merge-cachi2-sboms-script/test_merge_cachi2_sboms.py

```diff
@@ -44,6 +48,42 @@ def test_merge_sboms(data_dir: Path) -> None:
 
     assert json.loads(result) == expected_sbom
 
+    with open(f"{data_dir}/cachi2.bom.json") as f:
+        cachi2_sbom = json.load(f)
+
+    purls_taken_from_syft_sbom = get_purls(expected_sbom) - get_purls(cachi2_sbom)
+    assert sorted(purls_taken_from_syft_sbom) == [
```

Comment:
Does the `purls_taken_from_syft_sbom` need to be sorted? Could the compared hardcoded list be a set instead?

Reply:
Yup, changed to a set :+1: